components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/client/DefaultAuthenticationSeqMgtServiceClient.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.application.mgt.ui.client;
+
+import org.apache.axis2.client.Options;
+import org.apache.axis2.client.ServiceClient;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.xsd.DefaultAuthenticationSequence;
+import org.wso2.carbon.identity.application.mgt.defaultsequence.stub.IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException;
+import org.wso2.carbon.identity.application.mgt.defaultsequence.stub.IdentityDefaultSeqManagementServiceStub;
+import org.wso2.carbon.identity.application.mgt.ui.util.ApplicationMgtUIConstants;
+
+import java.rmi.RemoteException;
+
+/**
+ * Client for DefaultAuthSeqMgtService.
+ */
+public class DefaultAuthenticationSeqMgtServiceClient {
+
+    private IdentityDefaultSeqManagementServiceStub stub;
+    private static final Log log = LogFactory.getLog(DefaultAuthenticationSeqMgtServiceClient.class);
+    private final boolean debugEnabled = log.isErrorEnabled();
```

Comment:
why is debugenabled check is assigned to isErrorEnabled? shoudn't it be isDebugEnabled()?